src/api/catalog.ts

```diff
@@ -110,6 +111,36 @@ export default ({config, db}) => async function (req, res, body) {
   const s = Date.now()
   const reqHash = sha3_224(`${JSON.stringify(requestBody)}${req.url}`)
   const dynamicRequestHandler = () => {
+    // filter request parameters
```

Comment:
wdyt about moving all of this to separate function
```
function filterRequestParams (url, req, entityType, config) {
  if (config.elasticsearch.useRequestFilter && typeof config.entities[entityType] === 'object') {
  ...
    return `${urlParts[0]}?${querystring.stringify(urlParams)}`
  } else {
    return url
  }
}
```
and then fire it in `adjustBackendProxyUrl` ?
```
function adjustBackendProxyUrl (req, indexName, entityType, config) {
  ...
  url = filterRequestParams(url, entityType, config)
  return url
}
```

Reply:
sure, will do. That was a quickfix for one of the clients, so I've just copy/paste the solution